package.json

```diff
@@ -32,7 +32,7 @@
   },
   "files": [
     "src/",
-    "fake-timers.js"
+    "pkg/"
```

Comment:
Does this make it a breaking change? Are people possibly depending on this? @fatso83

Reply:
people could technically be importing the file by path, we could leave a file there for sole purpose of exporting the pkg/ one somehow i guess

Reply:
I think this is a breaking change, yes. On the other hand, I just merged another breaking change, so ... not an issue if both are released at the same time.